frontend/src/components/VaultList.vue

```diff
@@ -138,11 +138,7 @@ const ownsSelectedVault = computed(() => {
 
 const isAdmin = ref<boolean>();
 
-const filterOptions = {
-  accessibleVaults: t('vaultList.filter.entry.accessibleVaults'),
-  ownedVaults: t('vaultList.filter.entry.ownedVaults'),
-  allVaults: t('vaultList.filter.entry.allVaults'),
-};
+const filterOptions = ref< {[key: string]: string} >({accessibleVaults: t('vaultList.filter.entry.accessibleVaults'), ownedVaults: t('vaultList.filter.entry.ownedVaults')});
```

Comment:
I like the multiline version better (shouldn't our linter enforce it?)

Reply:
Applied in [fe12a83](https://github.com/cryptomator/hub/pull/230/commits/fe12a83fd2b435e14bc4d6fbcf277eccdbfb2265)